cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Compute Extent</title>
+</head>
+<body>
+<div id ="help">
+	<h4>General explanation</h4>
+	<p>
+The Compute Extent function uses the source data geometries to derive the geometric extent and map it to the target geometry.
+Three different options are available for extent computation: Bounding Box, Convex Hull and Union. Visual examples are 
+provided below.<br> 
+</br>
+<br> 
+</br>
+<strong>Bounding Box</strong><br>
+ </br>
+The bounding box option maps the smallest rectangle shape that enclose the source data geometry to the target data geometry. <br>
+</br>
```

Comment:
......that encloses.....        .......that enclose the source data geometries.....